assets/js/mbta.js

```diff
@@ -780,6 +773,57 @@ const stationConfig = {
       },
     },
   ],
+  Busway: [
+    {
+      id: 'SDUD',
+      name: 'Nubian',
+      zones: {
+        n: 'Platform F', s: 'Platform C', e: 'Platform D', w: 'Platform A', c: 'Platform E (East)', m: 'Platform E (West)',
+      },
+    },
+    {
+      id: 'GLEC',
+      name: 'Lechmere',
+      zones: {
+        n: 'North', s: 'South', e: false, w: false, c: false, m: false,
+      },
+    },
+    {
+      id: 'SHAR',
+      name: 'Harvard',
+      zones: {
+        n: 'Upper', s: false, e: false, w: false, c: false, m: 'Lower',
+      },
+    },
+    {
+      id: 'MMAT',
+      name: 'Mattapan',
+      zones: {
+        n: 'South', s: 'North', e: false, w: false, c: false, m: false,
```

Comment:
Is this due to ordering? Also, why do these have strings for values and others do not?

Reply:
nvm, i looked through`Station.js` and answered my questions

Reply:
It sounds like it's just due to the layout of the station being weird, per Dan's comment in the configuration:
```
// These zones are correct. The Mattapan South[bound] Busway shares a platform with 
// the northbound trolley, and vice versa.
```
Re: the text, you can either put `false` (no sign for this zone`, `true` (sign for this zone, just use the standard abbreviation), or a custom string to display. In this case NB / SB would be confusing, because in addition to the directions being swapped I believe what's being referred to here is the northern or southern part of the busway area, not the direction that buses from there are traveling.